tests/test_core.py

```diff
@@ -5202,6 +5202,10 @@ def test_fs_nodefs_home(self):
     self.emcc_args += ['-lnodefs.js']
     self.do_runf(test_file('fs/test_nodefs_home.c'), 'success', js_engines=[config.NODE_JS])
 
+  def test_fs_nodefs_stat(self):
```

Comment:
Given this test is specifically related to closure perhaps it should be named as such?    Maybe `test_fs_nodefs_errno_closure`?

Reply:
That's a better name for it; I've made the change in abd8d45c76faa110a894a36e94f4dd9ca2137c26

Reply:
> > Would updating `ERRNO_CODES` in src/library.js to be defined with strings for keys fix the issue also (or have the same effect)?
> 
> I just gave it a try, the test fails and the keys are transformed by Closure

I think its our library system the removes the quotes here, rather than closure.   Sadly I can't think of a better way to achieve this, but I worry that this will increase code side a fair bit for programs that need ERRNO_CODE.

I wonder if we can convert a lot of the uses of `ERRNO_CODES` to use the `{{{ cDefine('ENOTTY') }}}` pattern and therefore avoid including `ERRNO_CODES` unless really needed?

Reply:
Well, the problem is that the code is dynamically returned by Node itself as a property of the error, not minifying the codes is the only way I can think of to allow that dynamic lookup :/

For the code size, I'm unsure how big the real world impact could be, but I'm unsure if it'd have a big impact other than a couple bytes. In addition to that, I've seen during my Closure-ified code exploration that all references to the ERRNO_CODES enum were inlined; if it wasn't for that dynamic lookup imposed by nodefs.js the object would get stipped away.